web/src/pages/Route/components/Step1/RequestConfigView.tsx

```diff
@@ -388,15 +388,16 @@ const RequestConfigView: React.FC<RouteModule.Step1PassProps> = ({
         name='service_id'
       >
         <Select disabled={disabled}>
-          <Select.Option value=''>{}</Select.Option>
+          {/* NOTE value = '' mean no service_id select, need to find a better way */}
+          <Select.Option value='' key={Math.random().toString(36).substring(7)} >{null}</Select.Option >
           {serviceList.map(item => {
-            return <Select.Option value={item.id}>
+            return <Select.Option value={item.id} key={item.id}>
               {item.name}
             </Select.Option>
           })}
         </Select>
       </Form.Item>
-    </PanelSection>
+    </PanelSection >
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/103073875-5cc20f00-4603-11eb-9230-d13c53489efc.png)

what's this?

Reply:
maybe vscode auto format tools did this, let me recheck

Reply:
🤷‍♂️